contrib/githooks/README.md

```diff
@@ -0,0 +1,7 @@
+# Git hooks
+
+Installation:
```

Comment:
The user needs to install `gofmt`, etc. Can we mention that briefly here?

Reply:
I've added brief instructions on how to install `misspell` and `goimports`. `gofmt` comes out-of-the-box with the Go compiler.